addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2491,6 +2491,40 @@ options.registry.InnerChart = options.Class.extend({
     },
 });
 
+options.registry.ScrollButton = options.Class.extend({
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * Toggles the scroll down button.
+     */
+    toggleButton: function (previewMode, widgetValue, params) {
+        this.$target.find('.o_scroll_button').remove();
+        if (widgetValue) {
+            const button = document.createElement('button');
+            button.classList.add('o_scroll_button', 'fa', 'fa-angle-down');
+            button.title = _t("Scroll down to next section");
+            this.$target.append(button);
+        }
+    },
```

Comment:
I did not thinkg about color persistence but it could still be kept with simple addition here:
```javascript
this.$oldButton = this.$target.find('.o_scroll_button').detach();
if (widgetValue) {
    if (this.$oldButton && this.$oldButton.length) {
        this.$target.append(this.$oldButton);
    } else {
        ...
    }
},
```
It could actually make sense as the icon could also be changed through the editor.